### ISSUE TITLE
HTML Profile broken due to template location

### ISSUE THREAD

**Issue by rbt** [Raku/App-Rakubrew#17]
I think it might be looking in the wrong location for template.html

```
$ rakubrew version
Currently running moar-2020.06

$ raku --profile -e 'say "foo";'
foo
Writing profiler output to profile-1594494215.861275.html
Could not locate profiler/template.html; should have been at /rakudo-2020.06/install/share/nqp/lib/profiler/template.html; outputting sql data instead
Writing profiler output to profile-1594494215.861275.sql

$ find ~/.rakubrew -name template.html
/home/rbt/.rakubrew/versions/moar-2020.06/share/nqp/lib/profiler/template.html
```

**Comment by rbt** [comment]
This is a `rakubrew download` issue. `rakubrew build` works as expected.

**Comment by patrickbkr** [comment]
Actually this is a bug in NQP and it's (non-)handling of relocatability. I'm working on it.

**Comment by patrickbkr** [comment]
This is now fixed with [NQP 3e5403f1](https://github.com/Raku/nqp/commit/3e5403f161cc3c5b1a52c3670cc56aa80117a96c) and [Rakudo 37d5831](https://github.com/rakudo/rakudo/commit/37d58315d68ab9ea529b0cf27632ff973c8014f4). It should start working in Rakudo 2020.07.
